### PR TITLE
tests fixed

### PR DESCRIPTION
**Pull request by drakon660**
tests works on mac os (x86) and windows pc, sdks installed 6,7,8.

### CODE REVIEW

**Review by razvangoga — CHANGES_REQUESTED** [review]

File: src/Microsoft.Tye.Hosting/DockerRunner.cs [hunk 1]
```diff
@@ -64,7 +64,7 @@ public async Task StartAsync(Application application)
 
                 // Inject a proxy per non-container service. This allows the container to use normal host names within the
                 // container network to talk to services on the host
-                var proxyContainer = new DockerRunInfo($"mcr.microsoft.com/dotnet/sdk:8.0", "dotnet Microsoft.Tye.Proxy.dll")
+                var proxyContainer = new DockerRunInfo($"mcr.microsoft.com/dotnet/sdk:8.0", "dotnet --roll-forward Major Microsoft.Tye.Proxy.dll")
```
[the remark this answers is not in the document]
Comment (drakon660) [reply src/Microsoft.Tye.Hosting/DockerRunner.cs:67]: It should be refactor somehow, but now we must fix tests, I can add a TODO with info about rethinking and refactoring. Ok?

File: test/E2ETest/TyeRunTests.cs [hunk 2]
```diff
@@ -1050,9 +1051,9 @@ public async Task MultiRepo_WorksWithCloning()
 name: VotingSample
 services:
 - name: vote
-  repository: https://github.com/jkotalik/TyeMultiRepoVoting
+  repository: https://github.com/drakon660/TyeMultiRepoVoting
```
[the remark this answers is not in the document]
Comment (drakon660) [reply test/E2ETest/TyeRunTests.cs:1054]: will do

File: test/E2ETest/TyeRunTests.cs [hunk 3]
```diff
@@ -1050,9 +1051,9 @@ public async Task MultiRepo_WorksWithCloning()
...
 - name: results
-  repository: https://github.com/jkotalik/TyeMultiRepoResults";
+  repository: https://github.com/drakon660/TyeMultiRepoResults";
```
[the remark this answers is not in the document]
Comment (drakon660) [reply test/E2ETest/TyeRunTests.cs:1056]: will do

File: src/Microsoft.Tye.Hosting/DockerRunner.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (razvangoga) [reply src/Microsoft.Tye.Hosting/DockerRunner.cs:67]: Leave a TODO for the upgrade to dotnet 8.
I'll open a issue for it and link to this discussion if possible or add a screen shot if not so we don't forget about it

File: test/E2ETest/TyeRunTests.cs (hunk 3 above)
[the remark this answers is not in the document]
Comment (drakon660) [reply test/E2ETest/TyeRunTests.cs:1056]: same as above

File: test/E2ETest/TyeRunTests.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (drakon660) [reply test/E2ETest/TyeRunTests.cs:1054]: don't have access to push

File: src/Microsoft.Tye.Hosting/DockerRunner.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (drakon660) [reply src/Microsoft.Tye.Hosting/DockerRunner.cs:68]: updated

File: test/E2ETest/TyeRunTests.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (razvangoga) [reply test/E2ETest/TyeRunTests.cs:1054]: can you try again? there was some more access rights config stuff on the team that was missing - it should work now

**Review by razvangoga — APPROVED** [review]

**Pull request merged**